pkg/combustion/templates/13-embedded-registry.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+mkdir /opt/hauler
+
+mv haul.tar.zst /opt/hauler/haul.tar.zst
```

Comment:
It's better to template the input since a change to the constant will completely invalidate the script.